templates/base.html

```diff
@@ -29,12 +29,11 @@
                 {% activeurl %}
                 <ul class="row list-inline">
                     <li><a href="{% url 'home' %}" data-emoji-alt="🏡">Home</a></li>
-{#                    <li><a href="https://ti.to/djangocon-europe-2016/budapest/" data-emoji-alt="🎫">Tickets!</a></li>#}
-                                      <li><a href="/live/">Live!</a></li>
                     <li><a href="{% url 'schedule:schedule' %}" data-emoji-alt="🔈🔈🔈">Schedule</a></li>
                     <li><a href="{% url 'venue' %}" data-emoji-alt="🏠🎷🎵">Venue</a></li>
                     <li><a href="{% url 'blog:index' %}" data-emoji-alt="📰✍">Blog</a></li>
                     <li><a href="{% url 'faq:list' %}" data-emoji-alt="🙋🙋🙋">FAQ</a></li>
+                    <li><a href="{% url 'venue' %}" data-emoji-alt="📖">Glossary</a></li>
```

Comment:
Missed this one while reviewing, and before merging. It should have been `url 'glossary'`.